### PR TITLE
Tag FlatBuffers and upper bound Feather

### PR DESCRIPTION
**Pull request by quinnj**

### CODE REVIEW

File: Feather/versions/0.2.0/requires [hunk 1]
```diff
@@ -1,5 +1,5 @@
 julia 0.5
-FlatBuffers 0.1.1
+FlatBuffers 0.1.1 0.2.0
```
Comment (tkelman) [inline Feather/versions/0.2.0/requires]: unless these bounds are added to all past versions, Pkg may choose to downgrade Feather to a 0.1.x release in order to upgrade FlatBuffers.

Comment (quinnj) [inline Feather/versions/0.2.0/requires]: but not for Julia 0.5, right? All other past Feather versions were 0.4

Comment (tkelman) [inline Feather/versions/0.2.0/requires]: they have a min julia version but not a max julia version right? so the 1.x versions are allowed to be installed on 0.5

but yes if no FlatBuffers versions support 0.4 then we can't do the "fake dependency" trick

**Comment by quinnj** [comment]
Alright, updated all past Feather versions to have the upper bound on FlatBuffers 0.2.0

File: Feather/versions/0.1.2/requires [hunk 2]
```diff
@@ -1,5 +1,5 @@
 julia 0.4
-FlatBuffers
+FlatBuffers 0.1.1 0.2.0
```
Comment (tkelman) [inline Feather/versions/0.1.2/requires]: did these first versions need a lower bound?

Comment (quinnj) [inline Feather/versions/0.1.2/requires]: yes

Comment (tkelman) [inline Feather/versions/0.1.2/requires]: Feather was registered Jun 20th, but the 0.1.1 tag of FlatBuffers was not until August, how does the former depend on the latter if it didn't exist yet?

Comment (quinnj) [inline Feather/versions/0.1.2/requires]: Updated

**Pull request merged**